peps/pep-0798.rst

```diff
@@ -700,6 +713,65 @@ PEP.  As such, these forms should continue to raise a ``SyntaxError``, but with
 a new error message as described above, though it should not be ruled out as a
 consideration for future proposals.
 
+.. _pep798-alternativegenexpsemantics:
+
+Alternative Generator Expression Semantics
+------------------------------------------
+
+Another point of discussion centered around the semantics of unpacking in
```

Comment:
It might be worth showing some examples of what the actual difference in semantics is between `yield from` and the current approach with `for x in ...: yield x`. I think it matters if you `.send` something into the genexp?

Reply:
Yep, my understanding is that using `yield from` would mean that, if the thing we're unpacking is itself a generator, then anything we send to our top-level generator would actually go to the one we're unpacking.  With an explicit loop, that doesn't happen.  My gut feeling is that wanting to delegate to the thing being unpacked is probably a pretty niche situation (I'm not sure I know of a good practical use case for it), though I could certainly be wrong there.

Either way, I agree it's worth making that difference more explicit.  I'll add a paragraph here, but it might not be until tomorrow.  Let's hold off on merging until then.